website/azurerm.erb

```diff
@@ -359,6 +359,17 @@
                 </ul>
             </li>
 
+            <li<%= sidebar_current("docs-azurerm-resource-iothub") %>>
+              <a href="#">IotHub Resources</a>
```

Comment:
🤔 do you think it'd make sense for this to live under the `Messaging Resources` block?

Reply:
I agree. It shouldn't have its own block.